### PR TITLE
Sonar fixes 4523

### PR DESCRIPTION
**Pull request by hmcts-shajahan**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PRL-4523

### New line Sonar fix for following files ###
 
uk/gov/hmcts/reform/prl/controllers/FeeAndPayServiceRequestController.java 
src/main/java/uk/gov/hmcts/reform/prl/services/UploadAdditionalApplicationService.java

### Unnecessary throws exception removal for following files ###

uk/gov/hmcts/reform/prl/controllers/gatekeeping/ListOnNoticeController.java
uk/gov/hmcts/reform/prl/services/SendgridService.java

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by shashisk97 — APPROVED** [review]
approved

File: src/main/java/uk/gov/hmcts/reform/prl/services/BulkPrintService.java [hunk 1]
```diff
@@ -48,13 +48,7 @@ public UUID send(String caseId, String userToken, String letterType, List<Docume
...
         String s2sToken = authTokenGenerator.generate();
         final List<String> stringifiedDocuments = documents.stream()
-            .map(docInfo -> {
-                try {
-                    return getDocumentsAsBytes(docInfo.getDocumentBinaryUrl(), userToken, s2sToken);
-                } catch (IOException e) {
-                    throw new RuntimeException(e);
-                }
-            })
+            .map(docInfo -> getDocumentsAsBytes(docInfo.getDocumentBinaryUrl(), userToken, s2sToken))
```
Comment (AlokDatta) [inline src/main/java/uk/gov/hmcts/reform/prl/services/BulkPrintService.java:51]: you have missed the return keyword. double check the keyword, and test the code. verify with @shashisk97 once

**Review by AlokDatta — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/prl/services/BulkPrintService.java (hunk 1 above)
Comment (hmcts-shajahan) [reply src/main/java/uk/gov/hmcts/reform/prl/services/BulkPrintService.java:51]: As we discussed in the standup, return is not need.

File: src/main/java/uk/gov/hmcts/reform/prl/controllers/FeeAndPayServiceRequestController.java [hunk 2]
```diff
@@ -43,9 +43,12 @@ public class FeeAndPayServiceRequestController extends AbstractCallbackControlle
...
     public static final String CONFIRMATION_BODY_PREFIX_HELP_WITH_FEES = """
-        ### What happens next \n\n You will receive a confirmation email.
+        ### What happens next
+
+        You will receive a confirmation email.
         If the email does not appear in your inbox, check your junk or spam folder.
-        \n\n The court will review your help with fees application and tell you what happens next.
+
+        The court will review your help with fees application and tell you what happens next.
         """;
```
[the remark this answers is not in the document]
Comment (hmcts-shajahan) [reply src/main/java/uk/gov/hmcts/reform/prl/controllers/FeeAndPayServiceRequestController.java:52]: Resolved

File: src/main/java/uk/gov/hmcts/reform/prl/services/UploadAdditionalApplicationService.java [hunk 3]
```diff
@@ -641,7 +643,11 @@ public SubmittedCallbackResponse uploadAdditionalApplicationSubmitted(CallbackRe
             confirmationHeader = "# Application submitted";
             confirmationBody = """
                 You will get updates from the court about the progress of your application.
-                \n\n### What happens next \n\nThe court will review your documents and will be in touch to let you know what happens next.
+
+
+                ### What happens next
+
+                The court will review your documents and will be in touch to let you know what happens next.
```
[the remark this answers is not in the document]
Comment (hmcts-shajahan) [reply src/main/java/uk/gov/hmcts/reform/prl/services/UploadAdditionalApplicationService.java:650]: Resolved

File: src/main/java/uk/gov/hmcts/reform/prl/services/UploadAdditionalApplicationService.java [hunk 4]
```diff
@@ -628,7 +628,9 @@ public SubmittedCallbackResponse uploadAdditionalApplicationSubmitted(CallbackRe
             if (Yes.equals(caseData.getHwfRequestedForAdditionalApplications())) {
                 confirmationHeader = "# Help with fees requested";
                 confirmationBody = """
-                    ### What happens next \n\nThe court will review the document and will be in touch to let you
+                    ### What happens next
+
+                    The court will review the document and will be in touch to let you
```
[the remark this answers is not in the document]
Comment (hmcts-shajahan) [reply src/main/java/uk/gov/hmcts/reform/prl/services/UploadAdditionalApplicationService.java:633]: Resolved

File: src/main/java/uk/gov/hmcts/reform/prl/controllers/FeeAndPayServiceRequestController.java [hunk 5]
```diff
@@ -43,9 +43,12 @@ public class FeeAndPayServiceRequestController extends AbstractCallbackControlle
     public static final String CONFIRMATION_HEADER = "# Continue to payment";
     public static final String SERVICE_REQUEST_TAB = "#Service%20Request";
     public static final String CONFIRMATION_BODY_PREFIX_HELP_WITH_FEES = """
-        ### What happens next \n\n You will receive a confirmation email.
+        ### What happens next
+
+        You will receive a confirmation email.
```
[the remark this answers is not in the document]
Comment (hmcts-shajahan) [reply src/main/java/uk/gov/hmcts/reform/prl/controllers/FeeAndPayServiceRequestController.java:48]: Resolved

**Review by AlokDatta — APPROVED** [review]

**Pull request merged**